tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -26,6 +26,9 @@
 
 viewChangeTimeoutSec = 5
 
+loops = 4
+timeouts = 60 # variable accustoms to VM speed
```

Comment:
Can you explain a bit more how this "accustoms to VM speed" ? Is it an adaptable mechanism, or simply a time buffer?

Reply:
It's not adaptable per se. Tt just add this value to vurnable timeouts. U r right that commentis slightly misleading.